cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.28.0"
+__version__ = "2.28.1"
```

Comment:
```suggestion
__version__ = "2.27.1"
```

Reply:
ah no, 2.28 is already there

Reply:
2.28 already published, just changelog wasn't updated.